src/HttpServer.php

```diff
@@ -26,7 +26,7 @@ class HttpServer extends EventEmitter
     public function __construct(ServerInterface $socket, $callback)
     {
         if (!is_callable($callback)) {
-            throw new \Exception('The given parametr is not callable');
+            throw new \Exception('The given parameter is not callable');
```

Comment:
Will do. At the time of this merge request there is no `InvalidArgumentException`. Will come in with #6.